### ISSUE TITLE
Single organism mode: no genes shown on summary page

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1764]
In single user mode on the metagenotype-dev branch, I added some genes to my session and got:

![single-user-pub-page-bug-1](https://user-images.githubusercontent.com/90474/52328326-0ae65f00-2a54-11e9-8447-112acf40a9b7.png)

We shouldn't be mentioning "Pathogen organisms" and "Host Organisms" either.

**Comment by kimrutherford** [comment]
I was able to start an annotation by manually adding `/feature/gene/view/1` to the end of the URL.

**Comment by jseager7** [comment]
I don't think that table should even show in multi-organism mode. There's a conditional block in the Mason template that should prevent it:

https://github.com/pombase/canto/blob/616cc98d77d31f9e2f237138a1c555cb0a1803dc/root/curs/front_gene_section.mhtml#L22-L37

Is `$multi_organism_mode` being set correctly? It seems like there's a lot of other issues on the tracker similar to this that are related to faulty checks for multi-organism mode.

**Comment by kimrutherford** [comment]
I forgot to set `instance_organism` in the config.  The display is now correct in single organism mode.

> I don't think that table should even show in multi-organism mode.

I agree.  I think that needs more investigation so I'll leave this issue open and change the title.

Action item: try Canto in mutli-organism mode without pathogen/host host enabled.

**Comment by jseager7** [comment]
> I don't think that table should even show in multi-organism mode.

This was actually a typo, I meant 'single organism mode', but you're right. I've forgotten about checking Canto in multi organism mode, because these days I just see it as a synonym of pathogen&ndash;host mode, even though it's not.
